src/bindings/python/src/openvino/utils.py

```diff
@@ -38,6 +41,31 @@ def _add_openvino_libs_to_search_path() -> None:
                 os.add_dll_directory(os.path.abspath(lib_path))
 
 
+def get_cmake_path():
+    """Searches for the directory containing CMake files within the package install directory.
+
+    Parameters:
+    None
+
+    Returns:
+    str: The path to the directory containing CMake files, if found. Otherwise, returns empty string.
+    """
+    site_packages = chain(
+        (Path(__file__).parent.parent,),
+        site.getusersitepackages(),
+        site.getsitepackages()
+    )
```

Comment:
will it work for the following cases:
- OpenVINO is built to local directory and user set `PYTHONPATH` to `bin/intel64/Release/python` ? I suppose no, because this file does not know anything about OpenVINO binary dir with cmake files.
- OpenVINO is exposed via `setupvars.sh` ? I suppose `setupvars.sh` has higher priority because it sets env var, which has higher priority than cmake's `Package_DIR` env var?

Reply:
refactored as discussed offline